Resources/Server Info/Guidebook/Singularity.xml

```diff
@@ -0,0 +1,78 @@
+<Document>
+# Gravitational Singularity Engine
+
+Gravitational Singularity Engine can yield infinite power, no fueling required. It can also destroy whole station with equal ease. It uses a particle accelerator to fire high energy particles at a singularity generator to spawn a singularity. The singularity then pulses radiation which is absorbed by radiation collectors.
```

Comment:
```suggestion
Gravitational Singularity Engine can yield infinite power, no fueling required. It can also destroy the whole station with equal ease. It uses a particle accelerator to fire high energy particles at a singularity generator to form a singularity. The singularity then pulses radiation which is absorbed by radiation collectors.
```